doc/migration/v5.md

```diff
@@ -0,0 +1,184 @@
+# Angular Migration Guide v4 to v5
+
+## Major changes
+
+- Build system switched from pub transformers to build_runner/webdev. 
+  [Dart 2 Migration Guide for Web Apps](https://webdev-dartlang-org-dev.firebaseapp.com/dart-2)
+- Introduced Router 2.0. [Router Migration Guide](https://github.com/eredo/angular/blob/master/doc/router/migration.md)
```

Comment:
Could you change this URL to point to the upstream Angular repository instead of your fork? I noticed your fork is out-of-date because this doc is missing Dart syntax highlighting in all of the examples.